src/controllers/dlgprefcontroller.cpp

```diff
@@ -248,7 +259,6 @@ void DlgPrefController::enumeratePresets() {
     // user has their controller plugged in)
     m_ui.comboBoxPreset->addItem("...");
 
-    m_ui.comboBoxPreset->setInsertPolicy(QComboBox::InsertAlphabetically);
```

Comment:
Ah, I see -- yea PIE used to use a QMap with paths as keys, so that would have kept them sorted by filename (which is pretty close to sorting by preset name). Now that the QMap is gone, and assuming this insert policy thing never worked then there's nothing keeping them sorted.